crates/router/src/services/api.rs

```diff
@@ -450,7 +450,7 @@ async fn handle_response(
         .await
 }
 
-#[derive(Debug, Eq, PartialEq)]
+#[derive(Debug, Eq, PartialEq, Clone)]
```

Comment:
What's the need of the response to be clonable?

Reply:
Am using it here - https://github.com/juspay/hyperswitch/pull/1070/files#diff-9cffb95d59254b4103a363d800294d393f2ab62e7933a68b65904c90132be933R350

Reply:
Can't you clone only the `file_id` in that case? Something like this?

```rust
    let file_id = match create_file_response {
        services::ApplicationResponse::Json(ref res) => res.file_id.clone(),
        _ => Err(errors::ApiErrorResponse::InternalServerError)
            .into_report()
            .attach_printable("Unexpected response received from files create core")?,
    };
```